src/pages/User/components/PullRequests/index.js

```diff
@@ -126,6 +126,30 @@ export default class PullRequests extends Component {
     return count;
   }
 
+  getValidPullRequest(data) {
+    let validPullRequest = [];
+    data.items.forEach((pullRequest, index) => {
+      if (!pullRequest.labels.length) {
+        validPullRequest = [...validPullRequest, pullRequest]
+        return;
+      }
+
+      if (pullRequest.labels.length) {
```

Comment:
You have already checked the negative condition above `if (!pullRequest.labels.length) {` and it returns if holds true; so you could just skip this condition checking here.